codice/alliance
# [1.15.x] Add a configurable stream ID to the UDP stream monitor

#### What does this PR do?
Adds a configurable stream ID to the UDP stream monitor. This stream ID is added to the parent stream metacard and all chunk metacards.
#### Who is reviewing it? 
@clockard 
@sgalla 
#### How should this be tested?
Set up and start a UDP video stream with a custom stream ID. Inspect the created metacards and verify they all have the `video.stream.id` attribute with the custom stream ID you provided.
#### Checklist:
